schema/cache-v2.json

```diff
@@ -0,0 +1,177 @@
+{
+  "$schema": "http://json-schema.org/draft-07/schema#",
+  "$comment": "Version 1.0",
```

Comment:
The file is named `v2` and yet the version is `1.0` - isn't that inconsistent?

Reply:
Good point. Bumped the version in f1cd95fd6dc0bef0b85a339273d5495b94bd1db2